src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -41,6 +42,9 @@
           <item>
             <description>The transform supports scoring only one example at a time. However, retraining can be performed in batches.</description>
           </item>
+          <item>
+            <description>Advanced transfer learning/fine tuning scenarios are currently not possible due to lack of support for network/graph manipulation inside the model.</description>
```

Comment:
Can you be more specific? The line above says that retraining can be performed. Does this mean that you can retrain but not fine tune? Or do you mean something different than continued training? Maybe give an example: "Advanced transfer learning / fine tuning scenarios like ...".

Reply:
+1.  i think it might be good to be more specific here...

(fyi)  i created an issue #2348  which lists the limitations  for fine-tuning

---
In reply to: [253136068](https://github.com/dotnet/machinelearning/pull/2349#discussion_r253136068) [](ancestors = 253136068)